backend/open_webui/retrieval/vector/dbs/oracle23ai.py

```
@@ -523,7 +521,11 @@ def upsert(self, collection_name: str, items: List[VectorItem]) -> None:
                 raise
 
     def search(
-        self, collection_name: str, vectors: List[List[Union[float, int]]], limit: int
+        self,
+        collection_name: str,
+        vectors: List[List[Union[float, int]]],
+        filter: Optional[dict] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `filter`

(ARG002)